packages.json

```diff
@@ -1,6 +1,12 @@
 {
   "packages": {
     "plugins": [
+        {
+      "name": "Xcode Multi Edit",
```

Comment:
The name needs to match the `.xcodeproj` file, I don't think it will work this way :)